### ISSUE TITLE
Interline and alignment issues

### ISSUE THREAD

**Issue by Bokoblin** [Magisk-Modules-Repo/splatoon-2-font#1]
Hello, 

Default interline is too big, on my Nexus 6P with default font size, making the following visible issues :
* some parts of letters hidden (down-part of "y", "p",...) 
* text is not aligned with Radio buttons
* text is not centered in buttons

Changing font size in Android settings doesn't fix the issue.

BTW, I have seen in the source code that DPI is 320 by default, I don't know if it could fix the issue as 
enabling it, changing it, and re-archiving the zip causes Magisk Manager to not recognize the zip as a valid module.

Some example : 
![screenshot_20170818-103600 1](https://user-images.githubusercontent.com/8015512/29451683-de9dcf3e-8403-11e7-9e8a-6deb3cbab9f5.png)
![screenshot_20170818-103630 1](https://user-images.githubusercontent.com/8015512/29451665-cd9f16fc-8403-11e7-8fd7-d9f0d3839f51.png)

**Comment by ericchu94** [comment]
I can reproduce this issue, but I don't really have experience editing fonts.

If anyone wants to try to provide a fix, I can help test and package it

**Comment by Bokoblin** [comment]
I don't know either how to edit fonts, but I can look into it next week, as I also want to put a bigger difference between bold and normal weights

**Comment by Bokoblin** [comment]
[test1.zip](https://github.com/Magisk-Modules-Repo/splatoon-2-font/files/1254570/test1.zip)

I have changed some interline and offset values for 3 fonts just for you to try (as I can't repackage and flash on magisk manager -- it says package is not a magisk module) and tell me if it has changed something. Best case, the interline is reduced. Worst case, lines are not in correct order or are superposed.

**Comment by ericchu94** [comment]
![screenshot_20170827-233328](https://user-images.githubusercontent.com/7073910/29758396-371e8516-8b81-11e7-997f-8ac8a2a77c99.png)
![screenshot_20170827-233607](https://user-images.githubusercontent.com/7073910/29758395-371e884a-8b81-11e7-902a-90a8d8f3c002.png)
![screenshot_20170827-233948](https://user-images.githubusercontent.com/7073910/29758397-37241788-8b81-11e7-9438-f03f14106d04.png)

The first 2 screens look correct, but it looks like the gmail app has regressed.

**Comment by Bokoblin** [comment]
Okay I'll send you some others with all different values in a few hours

**Comment by Bokoblin** [comment]
[test2.zip](https://github.com/Magisk-Modules-Repo/splatoon-2-font/files/1257067/test2.zip)
Here is another zip of the font with 8 different values modified.

At least one of them should fit.
Once you've find the best one tell me which one.
I'll then try to change values for weight and to have condensed versions also.

**Comment by ericchu94** [comment]
Here are the screens for each change. I also packaged them if you want to try them yourself.
[Screenshots.zip](https://github.com/Magisk-Modules-Repo/splatoon-2-font/files/1257437/Screenshots.zip)
[modules.zip](https://github.com/Magisk-Modules-Repo/splatoon-2-font/files/1257440/modules.zip)

It looks like the best ones are Splatoon2-interline-800-and-offset-hhead-only and Splatoon2-offset-hhead-only, which are identical to me.

**Comment by Bokoblin** [comment]
I have chosen to keep Splatoon2-offset-hhead-only. Interline is near Roboto and the cutoff under a line has disappeared (down-part of "y", "p" visible).

**Comment by ericchu94** [comment]
Yes, currently all 18 fonts are the same. I will make a release with the Splatoon2-offset-hhead-only change.

If you think specific edits to bold fonts (or others) are needed, please feel free to do so.
